compiler/p/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -12175,7 +12175,9 @@
    /* .name        = */ "lxvl",
    /* .description =    "Load VSX Vector with Length", */
    /* .opcode      = */ 0x7C00021A,
-   /* .format      = */ FORMAT_XT_RA_RB_MEM,
+   // Even though lxvl is a memory instruction, its RB register is *not* an index register, so it
+   // should not be used with a TR::MemoryReference.
+   /* .format      = */ FORMAT_XT_RA_RB,
    // NOTE: This instruction was technically added in Power 9, but the Power 9 chips have
```

Comment:
Wondering why stxvl is not changed now.

Reply:
Oops, looks like this change got lost while resolving some merge conflicts. Fixed in ab506ae5547e29764eb2bcf1f5bc0d4794bd07e2.